gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -61,7 +115,7 @@ def __init__(self,
     self.formats = self._values_asdict(formats or {})
     self.contigs = self._values_asdict(contigs or {})
 
-  def update(self, to_merge):
+  def update(self, to_merge, force_merge_conflicts):
```

Comment:
We can have `force_merge_conflicts` default to False so that test cases do not need to specify it.

Reply:
Done!